CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ find_path(Artic_DIR NAMES artic-config.cmake PATHS ${Artic_DIR} $ENV{Artic_DIR}
 find_path(Impala_DIR NAMES impala-config.cmake PATHS ${Impala_DIR} $ENV{Impala_DIR} ${CMAKE_BINARY_DIR}/share/anydsl/cmake)
 
 set(AnyDSL_runtime_ENABLE_DEBUG_OUTPUT ${DEBUG_OUTPUT})
+set(AnyDSL_runtime_TARGET_NAME runtime CACHE STRING "Name of the cmake target for the AnyDSL runtime")
```

Comment:
I imagine your problem is collisions, but should we just rename the target to something more sensible like anydsl_runtime ? We can also use namespaces when exporting the target and name the .so/.dll something different

Reply:
This is partly the reason.

I definitely agree on renaming the default target name `runtime` to something else like `anydsl_runtime`. But in my case with Ignis I do need multiple versions of the runtime with different cmake flags. Which results in multiple cmake projects, but cmake namespaces is not really the issue for me.

Unfortunately, just renaming the output dll and similar is not sufficient due to embedded link paths inside .dll and .lib. Renaming the target at the configure stage fixes all of those.

PS: Renaming the .dll and adapting the .lib helps when linking the executable but not if dlopen is involved (like in my case)

Reply:
In fact, we should just introduce a namespace for the CMake targets. Then this will be "anydsl::runtime" by default.

Reply:
That sounds good. `anydsl::runtime` is a reasoable default name

Reply:
I like that idea.